mappings/net/minecraft/entity/vehicle/AbstractMinecartEntity.mapping

```diff
@@ -6,13 +6,19 @@ CLASS axl net/minecraft/entity/vehicle/AbstractMinecartEntity
 	METHOD <init> (Laij;Lbhh;)V
 		ARG 1 type
 	METHOD <init> (Laij;Lbhh;DDD)V
-		ARG 1 world
+		ARG 1 type
+		ARG 2 world
 		ARG 3 x
+		ARG 5 y
+		ARG 7 z
+	METHOD a (F)V
```

Comment:
uh, what is this?

Reply:
A method that sets some tracked data, where I tried to figure out what it exactly is, and then forgor to remove the parameter name.
It has something to do with the minecart shake animation, I am pretty sure, but there are 3 different types of that, and I gave up on it.